elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -612,18 +667,79 @@ static ResourceDefinition getTrustManagerDefinition() {
                 };
             }
 
-            private ValueSupplier<TrustManager> createX509CRLExtendedTrustManager(ServiceBuilder<TrustManager> serviceBuilder, OperationContext context, String algorithm, String providerName, InjectedValue<Provider[]> providersInjector, InjectedValue<KeyStore> keyStoreInjector, ModelNode crlNode, String aliasFilter) throws OperationFailedException {
-                String crlPath = PATH.resolveModelAttribute(context, crlNode).asStringOrNull();
-                String crlRelativeTo = RELATIVE_TO.resolveModelAttribute(context, crlNode).asStringOrNull();
-                int certPath = MAXIMUM_CERT_PATH.resolveModelAttribute(context, crlNode).asInt();
+            private ValueSupplier<TrustManager> createX509RevocationExtendedTrustManager(ServiceBuilder<TrustManager> serviceBuilder, OperationContext context, String algorithm, String providerName, InjectedValue<Provider[]> providersInjector, InjectedValue<KeyStore> keyStoreInjector, ModelNode crlNode, ModelNode ocspNode, Integer maxCertPath, String aliasFilter) throws OperationFailedException {
```

Comment:
Just a nitpick, optional - based on the latest change of `X509RevocationExtendedTrustManager` -> `X509RevocationTrustManager`, name of this method could also be updated.

Reply:
done